### DISCUSSION TITLE
SPAN reset/triggering a nonce exchange for Zniffer

### DISCUSSION THREAD

**Discussion by mundschenk-at** [zwave-js/zwave-js#7057] [Feature Requests]
### Description

It would be nice to trigger a manual SPAN reset for a specific node (or if necessary for all nodes) for use with the Zniffer feature. Right now it appears to depend mainly on luck whether one of the devices needs to do a SPAN re-synchronization while the Zniffer is active.

### Do you have a solution in mind?

An advanced action on the device in question (and maybe a second one for all devices on the controller).

**Comment by AlCalzone** [comment]
Possible workarounds for now:
- Power-cycle the device
- Execute this driver function (edit the node ID):
```ts
const nodeId = 2;
driver.getSecurityManager2(nodeId).deleteNonce(nodeId);
```

**Reply by mundschenk-at** [reply]
Thanks! (Power-cycling the device unfortunately is not really possible for power plugs, where I've had the issue mostly. At least not for the one with the computer that Z-Wave JS runs on.)
